MonoGame.Framework/Utilities/FuncLoader.Android.cs

```diff
@@ -15,7 +17,27 @@ internal class FuncLoader
 
         public static IntPtr LoadLibrary(string libname)
         {
-            return dlopen(libname, RTLD_LAZY);
+            // Let the OS search for the library by default.
+            IntPtr lib = dlopen(libname, RTLD_LAZY);
+            if (lib != IntPtr.Zero)
+            {
+                Console.WriteLine("FuncLoader.LoadLibrary {0}", libname);
```

Comment:
We should replace this with `Debug.WriteLine`.

Reply:
Will that work in shipping games?  I ask because ideally if a player or service provides a log from shipped device we would want to know if these fail to load.

Reply:
Hmm, I though you wanted this as debug information. I guess this is fine than.